clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,120 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 300}}
+{{$MEDIUM_BACKEND_SIZE := 150}}
+{{$SMALL_BACKEND_SIZE := 10}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
+name: ilbload
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: ILBConstantQPS
+  qpsLoad:
+    averageQps: {{$ilbQPS}}
+steps:
+# Create ILBs
+- measurements:
+  - Identifier: ServiceCreationLatency
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+  - Identifier: WaitForRunningDeployments
```

Comment:
The `WaitForControlledPodsRunning` is started but never used again. Do we need this?
If so, here is an example of gather method usage: https://github.com/kubernetes/perf-tests/blob/61f39edf3d70731cbf9225dc165ae103ec13e567/clusterloader2/testing/density/config.yaml#L160-L163

Reply:
Done